packages/gatsby-plugin-typescript/src/gatsby-node.js

```diff
@@ -14,6 +14,10 @@ function onCreateBabelConfig({ actions }, options) {
   actions.setBabelPlugin({
     name: require.resolve(`@babel/plugin-proposal-numeric-separator`),
   })
+  actions.setBabelPlugin({
+    name: require.resolve(`@babel/plugin-proposal-decorators`),
+    options: options && options.decoratorsOptions
```

Comment:
Needs a trailing comma.  Add Prettier to your editor to get automatic formatting for this, if you want.

Reply:
I've run `npm:format` and added both trailing commas (commits 2 & 3), but linting still fails.